templates/strings-dot-syntax-swift3.stencil

```diff
@@ -10,12 +10,14 @@ import Foundation
 // swiftlint:disable nesting
 // swiftlint:disable variable_name
 // swiftlint:disable valid_docs
+{% macro parametersBlock params %}{% for type in params.types %}_ p{{forloop.counter}}: {{type}}{% if not forloop.last %}, {% endif %}{% endfor %}{% endmacro %}
+{% macro argumentsBlock params %}{% for type in params.types %}p{{forloop.counter}}{% if not forloop.last %}, {% endif %}{% endfor %}{% endmacro %}
```

Comment:
Why not use Stencil's `|join(", ")` filter here instead of a `for` loop?
Would avoid the use of `if not forloop.last`…